core/src/epicli/data/common/ansible/playbooks/roles/backup/tasks/monitoring.yml

```diff
@@ -10,8 +10,11 @@
 - name: Provide parameters for the download_via_ssh backup copy method
   set_fact:
     backup_inline_script: >-
-      tar czpf - -C {{ snapshot_directory }}/ {{ snapshot_name }}/
-    backup_destination_file: "{{ backup_dir }}/mounted/prometheus_backup_{{ snapshot_name }}.tar.gz"
+      sudo tar czpf - -C {{ snapshot_directory }}/ {{ snapshot_name }}/
```

Comment:
Space before {{ snapshot_name }}. I'm not sure of this will not breake something

Reply:
Nope it's "-C {{ snapshot_directory }}"  and "{{ snapshot_name }}/", these are distinct arguments.   This causes the snapshot_directory path directory tree is not added to tar archive.